<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>OffsetDialog</class>
    <widget class="QDialog" name="OffsetDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>330</width>
                <height>158</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
                <rect>
                    <x>130</x>
                    <y>110</y>
                    <width>176</width>
                    <height>30</height>
                </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                    <widget class="QPushButton" name="cancelButton">
                        <property name="text">
                            <string>Cancel</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="okButton">
                        <property name="text">
                            <string>Ok</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
                <rect>
                    <x>40</x>
                    <y>50</y>
                    <width>261</width>
                    <height>31</height>
                </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                    <widget class="QLabel" name="label">
                        <property name="text">
                            <string>Enter Your Offset:</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QSpinBox" name="offsetBox"/>
                </item>
            </layout>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
